composer.json

```
@@ -28,7 +28,7 @@
     "ext-json": "*",
     "ext-pdo": "*",
     "typo3/cms-core": "^12.3.0",
-    "symfony/config": "^6.1"
+    "symfony/config": "^6.3 || ^7.0"
```

To be in line with the TYPO3 12LTS requirements, this should even be this:

```suggestion
    "symfony/config": "^6.4 || ^7.0"
```